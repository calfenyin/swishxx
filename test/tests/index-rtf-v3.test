index | | -e rtf:*.rtf -i rtf.index -r -v3 | . | 0
