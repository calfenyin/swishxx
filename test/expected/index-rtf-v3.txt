
.:
  info_group.rtf (38 words)
  sample.rtf (114 words)
  table_1.rtf (157 words)
  testrtf.rtf (317 words)

index: ranking index...
index: writing index...

index: done:
  4 indexed
  1533 words, 626 indexed, 229 unique

